build-settings/babel.config.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable import/no-commonjs */
+module.exports = {
+    presets: ["@babel/preset-flow", "@babel/preset-env"],
+    plugins: [
+        "@babel/plugin-proposal-class-properties",
+    ],
```

Comment:
https://babeljs.io/docs/en/babel-plugin-proposal-class-properties says that this plugin is included in @babel/preset-env now so maybe this isn't necessary.